### ISSUE TITLE
Clean Up in the datahub account

### ISSUE THREAD

**Issue by scott-yj-yang** [scott-yj-yang/LatePenalty#10]
Some modification has been made in the source code in the datahub of FA24 COGS108. Need to update that correspondingly to the package to make the changes persistent.

**Comment by scott-yj-yang** [comment]
Make changes in the followings:

```py
# calculate late days, use ReLU
        slip_day_used = adjusted_late_time.apply(lambda x: np.max([np.ceil(x.total_seconds()/60/60/24), 0])).apply(lambda x: x if x <= 5 else 0) # cap the maximum at 5
```

**Comment by scott-yj-yang** [comment]
Closed in PR#11
